dataland-frontend/src/components/pages/SearchTaxonomy.vue

```diff
@@ -2,47 +2,170 @@
   <AuthenticationWrapper>
     <TheHeader/>
     <TheContent>
-      <SearchTaxonomyHeader :scrolled="scrolled"/>
-      <EuTaxoSearchBar @autocomplete-focus="handleAutoCompleteFocus" :stockIndices="stockIndices" ref="euTaxoSearchBar"
-                       @scrolling="handleScrolling"/>
+      <SearchTaxonomyHeader :scrolled="pageScrolled"/>
+      <EuTaxoSearchBar v-if="!pageScrolled"
+                       ref="euTaxoSearchBar"
+                       @companyToQuery="handleCompanyQuery"/>
+      <MarginWrapper>
+        <IndexTabs v-if="!pageScrolled"
+                   ref="indexTabs"
+                   :initIndex="selectedIndex"
+                   @tab-click="toggleIndexTabs"/>
+      </MarginWrapper>
+      <div class="col-12 align-items-center grid bg-white d-search-toggle fixed" v-if="pageScrolled" >
+        <span class="mr-3 font-semibold">Search EU Taxonomy data</span>
+        <Button name="search_bar_collapse"
+                icon="pi pi-search" class="p-button-rounded surface-ground border-none m-2"
+                @click="toggleSearchBar">
+          <i class="pi pi-search" aria-hidden="true" style="z-index:20; color:#958D7C"/>
+        </Button>
+        <IndexTabs ref="indexTabs"
+                   :initIndex="selectedIndex"
```

Comment:
Change the look of the scrolled search bar and adjust it to the one at the top as discussed with Varenya and Leo in UX/UI meeting.